examples/flask/hello.py

```diff
@@ -82,6 +91,35 @@ def show_post(post_id):
 def get_tasks():
     return jsonify({'tasks': tasks})
 
+response = [
+    {
+        'id': 1,
+        'title': u'Buy groceries',
+        'description': u'Milk, Cheese, Pizza, Fruit, Tylenol',
+        'done': False
+    },
+    {
+        'id': 2,
+        'title': u'Learn Python',
+        'description': u'Need to find a good Python tutorial on the web',
+        'done': False
+    }
+]
+
+@app.route('/test/html_response', methods=['GET'])
+def html_response():
+    return """
```

Comment:
could we test and update the example with valid html body?

Reply:
Is valid html body like a html file?

Reply:
Changed